### DISCUSSION TITLE
Cloudflare Zone and Auth token

### DISCUSSION THREAD

**Discussion by ckvist72** [tobychui/zoraxy#221] [Q&A] [answered]
Hi. -. Whre do i find the zone and auth token in CloudFlare?

**Comment by tobychui** [comment]
@ckvist72  See https://github.com/tobychui/zoraxy/issues/215#issuecomment-2185394888 
I guess you do not need to fill in all inputs. Some can be left blank depending on use cases.

**Reply by ckvist72** [reply]
Yes. It seems the same. - But i still have problems.
In post https://github.com/tobychui/zoraxy/issues/215, it shows how to find the global key. - So now i have email and global key (aka. AuthKey)
But where do i find Zone ad Auth token? - I see an button, where i can generate a token. But is that Auth or Zone token? And where do i generate the other token?

**Reply by tobychui** [reply, marked as answer]
@ckvist72 What will happen if you left those two fields empty? 

As you can see from the #215, you only need either email + API key or the zone token for lego to works.
![圖片](https://github.com/tobychui/zoraxy/assets/24617523/9e849d35-be7e-4242-bc40-20fbafed3e5f)

**Reply by ckvist72** [reply]
Thanks. It was almost TOO simple. :-)
